server.go

```diff
@@ -807,8 +808,17 @@ func (s *Server) Start() {
 	}
 
 	// Initialize a gRPC connection for forwarding
-	var err error
-	s.grpcForwardConn, err = grpc.Dial(s.ForwardAddr, grpc.WithInsecure())
+	tlsConfig, err := s.Config.Tls.GetTlsConfig()
+	if err != nil {
+		s.logger.WithError(err).Fatal("failed to parse tls config")
+	}
+	if tlsConfig != nil {
+		s.grpcForwardConn, err = grpc.Dial(
+			s.ForwardAddr,
+			grpc.WithTransportCredentials(credentials.NewTLS(tlsConfig)))
+	} else {
```

Comment:
It's implied here that `tlsConfig` will be `nil` if... no config is specified? If no config is specified, will we not hit `err != nil` above?

Reply:
The `GetTlsConfig` returns `nil, nil` if no config is specified:
https://github.com/stripe/veneur/blob/master/util/tls/tls.go#L42-L44